packages/babel-preset-jsx/src/index.js

```diff
@@ -13,7 +13,7 @@ export default (_, {
   vOn = true,
   compositionAPI = false
 } = {}) => {
-  // compositionAPI: 'auto' | 'native' | 'plugin' | false
+  // compositionAPI: 'auto' | 'native' | 'plugin' | false | { importSource: string; }
```

Comment:
I agree that the object format is the most flexible.
But I think it's also worth adding a `vue-demi` shorthand here to simplify the configuration.

Reply:
well, i'll change the commit later